core/utils/type-utils/src/main/java/datawave/data/type/DateType.java

```diff
@@ -33,4 +46,16 @@ public String getDelegateAsString() {
     public long sizeInBytes() {
         return STATIC_SIZE + (2L * normalizedValue.length());
     }
```

Comment:
I think this needs to be updated to reflect the new object size?

Reply:
In the next commit